File: Python/Beginners/multiplos.py
# Caio Cesar 09/11/23
# BEE 1044

numeros = [int(n) for n in input().split()]

if numeros[0] % numeros[1] == 0 or numeros[1] % numeros[0] == 0:
    print('Sao Multiplos')
else:
    print('Nao sao Multiplos')

File: C++/Beginner/multiplos.cpp
// Caio Cesar 29/11/23
// BEE 1044

#include <iostream>

int main(){
    int num1, num2;

    scanf("%d %d", &num1, &num2);

    if((num1 % num2 == 0) || (num2 % num1 == 0)){
        printf("Sao Multiplos\n");
    }else{
        printf("Nao sao Multiplos\n");
    }

    return 0;
}